src/components/ArticlesTable/NotesPopup.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable react/jsx-no-bind */
+import { connect } from 'react-redux';
+
+import ReactMarkdown from 'react-markdown';
+
+import DeleteForeverIcon from 'material-ui/svg-icons/action/delete-forever';
+import { Component, PropTypes } from 'react';
+import Dialog from 'material-ui/Dialog';
+import Paper from 'material-ui/Paper';
+import TextField from 'material-ui/TextField';
+import IconButton from 'material-ui/IconButton';
+import RaisedButton from './../Fields/RaisedButton';
+import { createNote, updateNote, deleteNote } from './../../actions/notes';
+
+
+const inlineStyles = {
+  titleStyle: {
+    textAlign: 'center',
+    padding: '15px',
+  },
+  paperStyle: {
+    margin: '5px',
+    padding: '10px',
+  },
+  notePaper: {
+    margin: '5px',
+    padding: '10px',
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+};
+
+class NotesPopup extends Component {
+  static propTypes = {
+    articleId: PropTypes.number,
+    createNote: PropTypes.func,
+    deleteNote: PropTypes.func,
+    handleClose: PropTypes.func,
+    notes: PropTypes.array,
+    // updateNote: PropTypes.func,
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentNote: '',
+    };
+
+    [
+      'handleAddNoteClicked',
+      'handleDeleteNote',
```

Comment:
No need to bind `handleDeleteNote`.

Reply:
Why not?

Reply:
Change the scope of the function only if it is passed as a callback.

Reply:
OK, I'll try